src/lossfunctions.jl

```diff
@@ -113,6 +114,63 @@ end
 Returns a function to calculate scaled Gauss loss.
 Check the help of `scaled_gauss_aux`.
 """
-function ScaledGauss()
-    return scaled_gauss_aux
+function ScaledGauss(read_var=1)
+    return (µ, meas, storage=copy(µ)) -> scaled_gauss_aux(µ, meas, storage, read_var=read_var)
+end
+
+"""
+    anscombe_aux(μ, meas, storage=copy(μ))
+
+Calculates the Poisson loss using the Anscombe-based norm for `μ` and `meas`.
+`μ` can be of larger size than `meas`. In that case
+we extract a centered region from `μ` of the same size as `meas`.
+""" 
+function anscombe_aux(μ, meas, storage=copy(μ); b=1, sqrt_meas_b=0)
+    # due to numerical errors, μ can be negative or 0
+    mm = minimum(μ)
+    if mm <= 0
+        μ .= μ .+ eps(maximum(μ)) .+ abs.(mm)
+    end
+    storage .= abs2.(sqrt_meas_b .- sqrt.(μ .+ b))
+    return sum(storage)
+    #@tullio s = @inbounds abs2(sqrt_meas_b[i] - sqrt(μ[i] + b))
+    #return s
+end
+
+
+ # define custom gradient for speed-up
+ # ChainRulesCore offers the possibility to define a backward AD rule
+ # which can be used by several different AD systems
+function ChainRulesCore.rrule(::typeof(anscombe_aux), μ, meas, storage=copy(μ); b=1, sqrt_meas_b=0)
+    Y = anscombe_aux(μ, meas, storage; b=b, sqrt_meas_b=sqrt_meas_b)
+    function anscombe_aux_pullback(xbar)
+        # @show xbar
```

Comment:
remove pls

Reply:
Do you mean the comment with the show. Yes. remove